cli/src/main/java/org/jboss/as/cli/handlers/BaseOperationCommand.java

```diff
@@ -260,11 +269,16 @@ protected void addHeaders(CommandContext ctx, ModelNode request) throws CommandF
         opHeaders.set(headersNode);
     }
 
-    protected void handleResponse(CommandContext ctx, ModelNode response, boolean composite) throws CommandLineException {
+    protected void handleResponse(CommandContext ctx, OperationResponse response, boolean composite) throws CommandLineException {
```

Comment:
Please ping Alexey or Joe Wertz about this. I have the same concern here about an API change as I have with displayResponseHeaders, and this looks to be a bigger issue as the method is overridden quite a bit.

Perhaps add this method and leave the existing one, and then call to the existing one before calling handleAttachedFile. That's a more complicated API but doesn't break anyone. That's what I've done to evolve controller classes like AbstractAddStepHandler.